examples/kitchen-sink.yaml

```diff
@@ -91,6 +91,10 @@ logger_provider:
             #
             # Environment variable: OTEL_EXPORTER_OTLP_INSECURE, OTEL_EXPORTER_OTLP_LOGS_INSECURE
             insecure: false
+            # Configure retry policy
+            retry:
+              # Configure retry enabled
+              enabled: true
```

Comment:
If we buy my argument that `enabled: true` by default, then we should rename it to `disabled` to align with [spec naming conventions for booleans](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#boolean-value):

> All Boolean environment variables SHOULD be named and defined such that false is the expected safe default behavior.